docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -271,25 +261,57 @@ spec:
 ```
 
 
-## RBAC for Sessions
+## Teleport Resources
+
+RBACs lets teams limit what resources are available to teleport users. This can be
+helpful if you don't want regular users editing SSO `auth_connector` or creating and
+editing new roles `role`.
 
-As shown in the role example above, a Teleport administrator can restrict
-access to user sessions using the following rule:
+As list of all options is below.
```

Comment:
`As list of all options is below.` to `List of all options defined below.`?

Reply:
Thanks, Added.